index.html

```diff
@@ -4,276 +4,209 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title> Web developers Portfolio  </title>
+  <title>Portfolio</title>
   <link rel="stylesheet" href="style.css">
-  <link rel="stylesheet" href="script.js">
   <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
 </head>
 <body>
   
-  
     <header class="header">
-      <div class="menu-bar">
-        <a href="welcome" class="head">Welcome</a>
-        <nav class="menu-bar2">
-          <ul class="nav-menu" id="menu">
-            <a href="" onclick="" class="close-button">&times;</a>
-            <li ><a onclick="closeMenu()" href="#portfolio">Portfolio</a></li>
-            <li><a onclick="closeMenu()" href="#about">About</a></li>
-            <li><a onclick="closeMenu()" href="#contact">Contact</a></li>
-          </ul>
-          <div class="hamburger">
-            <span class="bar"></span>
-            <span class="bar"></span>
-            <span class="bar"></span>
-          </div>
-          </nav>
-      </div>
-        <nav class="menu-bar-desktop">
-          <ul class="nav-menu-desktop">
-            <!-- <a href="" onclick="" class="close-button">&times;</a> -->
-            <li ><a href="#portfolio">Portfolio</a></li>
-            <li><a href="#about">About</a></li>
-            <li><a href="#contact">Contact</a></li>
-          </ul>
-        </nav>
-        
+      <!-- <a href="welcome" class="head">Welcome</a> -->
```

Comment:
Nice work so far. However, as part of HTML/CSS best practices, do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.
Kindly consider removing the commented line of code from your codebase. :pray: